content/en/docs/tasks/debug/debug-cluster/_index.md

```diff
@@ -14,7 +14,16 @@ problem you are experiencing. See
 the [application troubleshooting guide](/docs/tasks/debug/debug-application/) for tips on application debugging.
 You may also visit the [troubleshooting overview document](/docs/tasks/debug/) for more information.
 
+ Before upgrading to Kubernetes 1.24 , please go through [Urgent Upgrade Notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md#urgent-upgrade-notes).
+
 <!-- body -->
+## Cluster not starting
+
+*If you get the following error after upgrading to Kubernetes 1.24, and you were previously using Docker Engine as your container runtime, go through the [Migrating from dockershim](/docs/tasks/administer-cluster/migrating-from-dockershim/) guide.*
+
+```
+Error: failed to parse kubelet flag: unknown flag: --network-plugin
+```
```

Comment:
I'd move this to be a level-3 heading / section within “Cluster failure modes”, and make the heading be similar to one of:

- “Retaining old settings after cluster upgrade”
- “Incomplete upgrade from Kubernetes v1.23 or earlier”
- “Node configuration wrong after switching from Docker Engine container runtime”

What do you think @NitishKumar06 ?

Reply:
This sounds good @sftim . Correct me if I'm wrong: 

So we can just transfer the above newly added information under "CLUSTER FAILURE MODES" using level-3 heading like [Specific scenarios](https://kubernetes.io/docs/tasks/debug/debug-cluster/#contributing-causes) with the above heading title which suits best. Right?